### ISSUE TITLE
EmptyLinesAroundClassBody and EmptyLinesAroundModuleBody are different from the Intercom Style

### ISSUE THREAD

**Issue by wiracocha** [intercom/intercom-rubocop#2]
We are now adding empty lines in classes and module which we never had before. See Intercom Style guide examples.

**Comment by eugeneius** [comment]
Where did this preference come from? It's the opposite of the [RuboCop default](https://github.com/bbatsov/rubocop/blob/v0.47.1/config/default.yml#L444).

**Comment by wiracocha** [comment]
Yeah there're other defaults that have changed as well. I'm not sure why we changed them (Like I never seen any discussion in engineering)

**Comment by brandonweiss** [comment]
Hey! We actually had both ways, because we weren't enforcing either.

The Intercom style guide is just a fork of the Ruby community style guide which is where most RuboCop defaults originally came from. The RuboCop defaults aren't necessarily default because they're generally preferred—a lot of the defaults are default because when a potentially better option was proposed, it was added as an option but the default wasn't switched because that would make everyone's linter throw offenses when they updated. So the onus is on the consumer to be aware of new settings and adjust their configuration accordingly.

The intention is that this supplants the Intercom Ruby style guide and is better than it—a living style guide that can proactively guide people rather than having to be read and then memorized beforehand. 

In this particular style, we looked at the two options we used and decided that having empty lines around the class and module body seemed to be preferred because it improved readability. It's similar to the reason we add whitespace between method definitions.

There wasn't a discussion in engineering because we didn't want to needlessly bikeshed on each one if most people would agree with it. Especially since it's opt-in. The few that have deviations we arrived at with a compelling, logical argument.

If we want we can change this, similar to how we changed single quotes to double quotes. We just a better, compelling logical argument the other way.

Thoughts?

**Comment by wiracocha** [comment]
Vox populi, vox dei (The voice of the people is the voice of the god). 

>  we used and decided that having empty lines around the class and module body seemed to be preferred because it improved readability

Who is `we` and where the `seemed to be preferred` comes from? I open randomly files in our codebase and I don't see empty lines around modules or classes. Same for example if I open the codebase for mutations or rails or other gems. So I don't see the empty lines being a preferred option. 

And I agree, If empty lines around classes need to be added, where is the compelling logical argument? 

We need to stick to defaults, because those defaults is what ruby developers expect. If they need to change we need a discussion for that, because in fairness, the defaults are already good. Why change expectations or something that is already working?
